arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -74,6 +75,30 @@ RestStatus RestDocumentHandler::execute() {
   return RestStatus::DONE;
 }
 
+void RestDocumentHandler::shutdownExecute(bool isFinalized) noexcept {
+  TRI_DEFER(RestVocbaseBaseHandler::shutdownExecute(isFinalized));
```

Comment:
can be changed from a TRI_DEFER to a simple call of `shutdownExecute`, and can be moved behind the try..catch block.